app/forms/application_form.rb

```diff
@@ -0,0 +1,26 @@
+class ApplicationForm
+  include ActiveModel::Model
+  include ActiveModel::Naming
+
+  validates :body, :user_id, presence: true
+
+  def save
+    update_form_attributes
```

Comment:
It's a really really bad idea call method that does not define in this class or classes this class inherited.